src/style.css

```diff
@@ -1 +1,46 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-repeat: no-repeat ;
+  background-position: center center;
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
```

Comment:
<img width="475" alt="image" src="https://github.com/mate-academy/layout_stars/assets/86789161/44acc63a-5c26-40b2-924d-413eea32170d">

Reply:
Do something similar to this